RELEASING.md

```diff
@@ -24,95 +24,211 @@ tools must be installed.
+- [ ] Tag release
+- [ ] Draft release and upload artifacts
+- [ ] Upload to sonatype
+- [ ] Document lessons learned
+
 Release Steps
 -------------
 
 ### Preparing Documentation
 
-Update version-sensitive files in the root and documentation sources
-in the `docs` subdirectory:
+Update version-sensitive files in the root and documentation sources in the `docs` subdirectory:
 
-* `docs/changelog.txt`
-* `docs/versions.txt`
-* `docs/upgrade.txt`
+* `docs/changelog.adoc`
+* `docs/versions.adoc`
+* `docs/upgrade.adoc`
 
-Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to
-build a set of docs for this release and copy them to the cloned
-`docs.janusgraph.org` repo which you will update later.
+Use the [`docs/build-and-copy-docs.sh`](docs/build-and-copy-docs.sh) script to build a set of docs for this release and copy them to the cloned `docs.janusgraph.org` repo which you will update later.
+**NOTE:** the `docs/build-and-copy-docs.sh` script by default expects the janusgraph and docs.janusgraph.org repositories to be located at `~/github/janusgraph/janusgraph` and `~/github/janusgraph/docs.janusgraph.org` respectively.
+It is important that the the `latest` symbolic link at the root of docs.janusgraph.org is pointing to the correct version as well.
 
-You may also need to update the following files in the main repo for any new
-or updated dependencies:
+You may also need to update the following files in the main repo for any new or updated dependencies:
 
 * `NOTICE.txt`
 
-### Preparing the Local Repository
-
-Commit or stash any uncommitted changes.
-
-Recommended but not required:
-
-* Delete untracked files and directories in the clone by running
-  <code>git clean -fd</code>
-
-### Deploying Artifacts & Archives
-
-```bash
-# This script does several things.
-#
-# * Prompts for a ${janusgraph.compatible.versions} update
-# * Locally commits the release using the release plugin
-# * Deploys Maven artifacts to Sonatype OSS (staging, not released yet)
-# * Uploads zipfiles to S3
-# * Locally commits gh-pages updates (index.html and javadocs)
-# * Uploads AsciiDoc-generated documentation to S3
-#
-# Although it uploads to Sonatype OSS Staging and S3, it does
-# not push to github nor does it release the Sonatype repo.
-# This step is still essentially reversible: just destroy the
-# local commit history, drop the Sonatype OSS Staging repo, and
-# delete the files uploaded to S3.
-janusgraph-dist/src/release/release.sh
+### Testing
+By default Travis CI only runs tests against a single version of Cassandra.
+In order to run compatibility testing on all version of Cassandra listed as supported you can add [full build] to you commit message.
+
+### Versions
+
+This command will create a commit that updates the version from `x.y.z-SNAPSHOT` to `x.y.z`.
+The current `pom.xml` files will be backed up with the name `pom.xml.releaseBackup` and left in staging.
+
+```Shell
+mvn clean release:prepare
```

Comment:
I am not sure but maybe we could also mention that this command have some problems and last releases are performed without this command? I.e. manually.

Reply:
I talk again with Jason and he did in fact use this command when he did his releases. I tried it on 0.3 and it worked when I checked out the commit before the 0.3.2 release commit. I can try against the revert commit on Monday.

Reply:
I will also try this command again when preparing release artifacts.